.github/labels.yml

```diff
@@ -138,11 +138,6 @@
   description: "Make this PR count for hacktoberfest"
   color: "ff7518"
 
-# This label can be added to accept PRs as part of Hacktoberfest
-- name: "hacktoberfest-accepted"
```

Comment:
I'd like to keep this, for the same reasons as mentionen in exercism/erlang-analyzer#35

Reply:
See https://github.com/exercism/erlang-analyzer/pull/35#discussion_r821385960

Reply:
See https://github.com/exercism/erlang-analyzer/pull/35#discussion_r821385960